code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -548,7 +571,96 @@ GLOBAL_LIST_EMPTY(PDAs)
+							var/howmuch = input(usr, "How much do you want to transfer?", "Amount", 0) as num
+							if(!howmuch || howmuch < 0)
+								return
+							if(id && !id.registered_account.has_money(howmuch))
+								to_chat(usr, "<span class='warning'>The linked account doesn't have the funds for that.</span>")
+								return
+							if(id && id.registered_account.has_money(howmuch)) //prevents runtimes
+								B.adjust_money(howmuch)
+								B.log_money(howmuch,"[id.registered_account.account_id] Account")
+								B.bank_card_talk("Transfer of $[howmuch] received from [id.registered_account.account_id]",TRUE)
+								id.registered_account.adjust_money(-howmuch)
+								id.registered_account.log_money(-howmuch,"[B.account_id] Account")
+								to_chat(usr, "<span class='notice'>You successfully transfered $[howmuch] from account [id.registered_account.account_id] to [B.account_id]</span>")
+								return
+					to_chat(usr, "<span class='warning'>The account ID number provided is invalid.</span>")
+				else if(id)
+					to_chat(usr, "<span class='notice'>There is no registered account on the ID inserted.</span>")
+				else if(!id)
+					to_chat(usr, "<span class='notice'>There is no ID inserted.</span>")
+				
+			if("Eject Holocredits")
+				if(id.registered_account)
+					var/amount_to_remove = input(usr, "How much do you want to withdraw?", "Account Reclamation", 5) as num
+					if(!amount_to_remove || amount_to_remove < 0)
+						return
+					if(id.registered_account.adjust_money(-amount_to_remove))
+						var/obj/item/holochip/holochip = new (usr.drop_location(), amount_to_remove)
+						id.registered_account.log_money(-amount_to_remove,"Holochip Withdraw")
+						usr.put_in_hands(holochip)
+						to_chat(usr, "<span class='notice'>You withdraw [amount_to_remove] credits into a holochip.</span>")
+						return
+					else
+						to_chat(usr, "<span class='warning'>The linked account doesn't have the funds for that.</span>")
+				else if(id)
+					to_chat(usr, "<span class='notice'>There is no registered account on the ID inserted.</span>")
+				else if(!id)
+					to_chat(usr, "<span class='notice'>There is no ID inserted.</span>")
+					
+			if("Create Account")
+				if(id && !id.registered_account)
+					if(!id.assignment)
+						to_chat(usr, "<span class='notice'>Nanotrasen bank requires a registered job to create an account</span>")
+						return
+					if(!id.registered_name)
+						to_chat(usr, "<span class='notice'>Nanotrasen bank requires a registered name to create an account</span>")
+						return
+					if(iscarbon(usr))
+						var/list/all_jobs = subtypesof(/datum/job)
+						var/datum/job/assign = null
+						to_chat(usr, "<b>wat.</b>")
+						for(var/datum/job/J in all_jobs)
+							to_chat(usr, "<b>[id.assignment] and [J.title].</b>")
+							if(J.title == id.assignment)
+								assign = J
+								break
+						if(!assign)
+							assign = id.assignment
+						var/datum/bank_account/A = new (id.registered_name,assign)

```

Comment:
I need help here. The `id.assignment` returns the job title, like "Assistant" for example. I need to convert this to `/datum/job/assistant`, otherwise it gives a runtime every payday

Also, should this new created accounts receive a paycheck? Should custom jobs receive paycheck?